src/bag3_testbenches/measurement/mos/sim.py

```diff
@@ -580,15 +607,11 @@ async def async_measure_performance(self, name: str, sim_dir: Path, sim_db: Simu
                 res['ss_file'] = ss_fname
 
             elif tbm_name == 'noise':
-                # TODO: needs to be verified
-                tbm: MOSNoiseTB
-                temp = self.specs['noise_temp_kelvin']
-                fstart = self.specs['noise_integ_fstart']
-                fstop = self.specs['noise_integ_fstop']
-                scale = self.specs.get('noise_integ_scale', 1.0)
-
                 ss_params = load_sim_file(ss_fname)
-                ss_params = tbm.get_integrated_noise(data, ss_params, temp, fstart, fstop, scale=scale)
+                temp = 273 + float(tbm.sim_envs[0].split('_')[1])
+                # TODO: should frequency range of gamma calculation be different from
```

Comment:
Technically gamma should be calculated by using the data points after the flicker noise corner. Feel free to keep it as TODO and maybe we can implement later if necessary.